Assets/Scripts/Rocket/Rocket.cs

```diff
@@ -51,6 +51,6 @@ private void OnCollisionEnter2D(Collision2D collision)
     private IEnumerator EnableColliderAfter(float time)
     {
         yield return new WaitForSeconds(time);
-        collider.enabled = true;
+        Physics2D.IgnoreCollision(parentPlayer.GetComponent<Collider2D>(), collider, false);
```

Comment:
How about making the collider of the parent player a public variable? Like parentPlayer.collider instead